packages/office-ui-fabric-react/src/components/Persona/Persona.scss

```diff
@@ -69,26 +69,23 @@ $ms-Persona-presenceSizeXl: 28px;
   line-height: 1;
   position: relative;
   height: $ms-Persona-sizeMd;
-  display: table;
-  table-layout: fixed;
-  border-collapse: separate;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: flex-start;
 
   .ms-ContextualHost {
     display: none;
   }
 }
 
 .ms-Persona-imageArea {
-  position: absolute;
+  position: relative;
   overflow: hidden;
   text-align: center;
-  max-width: $ms-Persona-sizeMd;
-  height: $ms-Persona-sizeMd;
+  flex: 0 0 $ms-Persona-sizeMd;
   border-radius: 50%;
   z-index: $ms-zIndex-back;
```

Comment:
Can you also try removing z-index here? Might not need it anymore since we've removed position:absolute and thus nothing is layering on top of something else.